user/src/test/java/com/jumar/user/utils/PasswordUtilsTest.java

```diff
@@ -0,0 +1,29 @@
+package com.jumar.user.utils;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PasswordUtilsTest {
+
+    @Test
+    void should_returnTrue_when_comparingTwoHashesOfSamePassword() throws Exception{
+        String password = "passwordTest";
```

Comment:
again, this is not testing business logic, really just a test of the hashing algorithm but we wouldn't write a test for that.